### PR TITLE
Add better-ajv-errors w/ npm auto-update

### LINKED ISSUE #1721: Add: better-ajv-errors
### Library name

better-ajv-errors

### Library license

https://github.com/atlassian/better-ajv-errors/blob/main/LICENSE

### Library source

https://www.npmjs.com/package/better-ajv-errors

### Files to include

lib/cjs/index.js
lib/cjs/validation-errors/*.js
lib/cjs/json/*.js

### PR DESCRIPTION
**Pull request by jeremyfiel**
closes #1721

### CODE REVIEW

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
🤔 It looks like you're including the cjs version here that uses Node.js `require` calls that won't work in a browser? Potentially we could add the ESM version for folks that're using modules, but ideally there'd be a pre-bundled browser version?

**Comment by jeremyfiel** [comment]
Ya, all of these packages I added as a user.  So I don't have  any way to add the bundling.

**Comment by MattIPv4** [comment]
If this is switched over to the ESM version I'd be happy to add it for folks that're using browser modules, or consuming this through runtimes like Deno etc.

**Comment by jeremyfiel** [comment]
Ok will update it

**Comment by jeremyfiel** [comment]
ok, i think that glob pattern will work.  thanks

File: packages/b/better-ajv-errors.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "license": "Apache-2.0",
+  "filename": "index.mjs",
+  "autoupdate": {
+    "source": "npm",
+    "target": "better-ajv-errors",
+    "fileMap": [
+      {
+        "basePath": "lib/esm",
+        "files": [
+          "/**/*.@(mjs?(.map))"
```
Comment (MattIPv4) [inline packages/b/better-ajv-errors.json:26]:
```suggestion
          "**/*.mjs?(.map)"
```

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**